Crash on sample app

Hi, when I start SpeechRecognizer or Audio Recording i got a crash direct.
```
FATAL EXCEPTION: main
  Process: com.cleveroad.audiovisualization.example, PID: 19197
  Theme: themes:{}
  android.view.InflateException: Binary XML file line #8: Binary XML file line #8: Error inflating class com.cleveroad.audiovisualization.GLAudioVisualizationView
	  at android.view.LayoutInflater.inflate(LayoutInflater.java:539)
	  at android.view.LayoutInflater.inflate(LayoutInflater.java:423)
	  at com.cleveroad.example.SpeechRecognitionFragment.onCreateView(SpeechRecognitionFragment.java:35)
	  at android.support.v4.app.Fragment.performCreateView(Fragment.java:1974)
	  at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1067)
	  at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1252)
	  at android.support.v4.app.BackStackRecord.run(BackStackRecord.java:742)
	  at android.support.v4.app.FragmentManagerImpl.execPendingActions(FragmentManager.java:1617)
	  at android.support.v4.app.FragmentManagerImpl$1.run(FragmentManager.java:517)
	  at android.os.Handler.handleCallback(Handler.java:739)
	  at android.os.Handler.dispatchMessage(Handler.java:95)
	  at android.os.Looper.loop(Looper.java:148)
	  at android.app.ActivityThread.main(ActivityThread.java:5456)
	  at java.lang.reflect.Method.invoke(Native Method)
	  at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
	  at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)
   Caused by: android.view.InflateException: Binary XML file line #8: Error inflating class com.cleveroad.audiovisualization.GLAudioVisualizationView
	  at android.view.LayoutInflater.createView(LayoutInflater.java:645)
	  at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:764)
	  at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:704)
	  at android.view.LayoutInflater.rInflate(LayoutInflater.java:835)
	  at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:798)
	  at android.view.LayoutInflater.inflate(LayoutInflater.java:515)
	  at android.view.LayoutInflater.inflate(LayoutInflater.java:423) 
	  at com.cleveroad.example.SpeechRecognitionFragment.onCreateView(SpeechRecognitionFragment.java:35) 
	  at android.support.v4.app.Fragment.performCreateView(Fragment.java:1974) 
	  at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1067) 
	  at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1252) 
	  at android.support.v4.app.BackStackRecord.run(BackStackRecord.java:742) 
	  at android.support.v4.app.FragmentManagerImpl.execPendingActions(FragmentManager.java:1617) 
	  at android.support.v4.app.FragmentManagerImpl$1.run(FragmentManager.java:517) 
	  at android.os.Handler.handleCallback(Handler.java:739) 
	  at android.os.Handler.dispatchMessage(Handler.java:95) 
	  at android.os.Looper.loop(Looper.java:148) 
	  at android.app.ActivityThread.main(ActivityThread.java:5456) 
	  at java.lang.reflect.Method.invoke(Native Method) 
	  at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726) 
	  at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616) 
   Caused by: java.lang.reflect.InvocationTargetException
	  at java.lang.reflect.Constructor.newInstance(Native Method)
	  at android.view.LayoutInflater.createView(LayoutInflater.java:619)
	  at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:764) 
	  at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:704) 
	  at android.view.LayoutInflater.rInflate(LayoutInflater.java:835) 
	  at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:798) 
	  at android.view.LayoutInflater.inflate(LayoutInflater.java:515) 
	  at android.view.LayoutInflater.inflate(LayoutInflater.java:423) 
	  at com.cleveroad.example.SpeechRecognitionFragment.onCreateView(SpeechRecognitionFragment.java:35) 
	  at android.support.v4.app.Fragment.performCreateView(Fragment.java:1974) 
	  at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1067) 
	  at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1252) 
	  at android.support.v4.app.BackStackRecord.run(BackStackRecord.java:742) 
	  at android.support.v4.app.FragmentManagerImpl.execPendingActions(FragmentManager.java:1617) 
	  at android.support.v4.app.FragmentManagerImpl$1.run(FragmentManager.java:517) 
	  at android.os.Handler.handleCallback(Handler.java:739) 
	  at android.os.Handler.dispatchMessage(Handler.java:95) 
	  at android.os.Looper.loop(Looper.java:148) 
	  at android.app.ActivityThread.main(ActivityThread.java:5456) 
	  at java.lang.reflect.Method.invoke(Native Method) 
	  at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726) 
	  at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616) 
   Caused by: android.content.res.Resources$NotFoundException: Array resource ID #0x0
	  at android.content.res.Resources.obtainTypedArray(Resources.java:608)
	  at com.cleveroad.audiovisualization.GLAudioVisualizationView$Configuration.<init>(GLAudioVisualizationView.java:153)
	  at com.cleveroad.audiovisualization.GLAudioVisualizationView.<init>(GLAudioVisualizationView.java:36)
	  at java.lang.reflect.Constructor.newInstance(Native Method) 
	  at android.view.LayoutInflater.createView(LayoutInflater.java:619) 
	  at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:764) 
	  at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:704) 
	  at android.view.LayoutInflater.rInflate(LayoutInflater.java:835) 
	  at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:798) 
	  at android.view.LayoutInflater.inflate(LayoutInflater.java:515) 
	  at android.view.LayoutInflater.inflate(LayoutInflater.java:423) 
	  at com.cleveroad.example.SpeechRecognitionFragment.onCreateView(SpeechRecognitionFragment.java:35) 
	  at android.support.v4.app.Fragment.performCreateView(Fragment.java:1974) 
	  at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1067) 
	  at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1252) 
	  at android.support.v4.app.BackStackRecord.run(BackStackRecord.java:742) 
	  at android.support.v4.app.FragmentManagerImpl.execPendingActions(FragmentManager.java:1617) 
	  at android.support.v4.app.FragmentManagerImpl$1.run(FragmentManager.java:517) 
	  at android.os.Handler.handleCallback(Handler.java:739) 
	  at android.os.Handler.dispatchMessage(Handler.java:95) 
	  at android.os.Looper.loop(Looper.java:148) 
	  at android.app.ActivityThread.main(ActivityThread.java:5456) 
	  at java.lang.reflect.Method.invoke(Native Method) 
	  at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726) 
	  at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616) 
```

--- a/library/src/main/java/com/cleveroad/audiovisualization/GLAudioVisualizationView.java
+++ b/library/src/main/java/com/cleveroad/audiovisualization/GLAudioVisualizationView.java
@@ -146,7 +146,7 @@ public class GLAudioVisualizationView extends GLSurfaceView implements AudioVisu
                 if (bgColor == Color.TRANSPARENT) {
                     bgColor = ContextCompat.getColor(context, R.color.av_color_bg);
                 }
-                int arrayId = array.getResourceId(R.styleable.GLAudioVisualizationView_av_wavesColors, 0);
+                int arrayId = array.getResourceId(R.styleable.GLAudioVisualizationView_av_wavesColors, R.array.av_colors);
                 if (isInEditMode) {
                     colors = new int[layersCount];
                 } else {

